build-package.sh

```diff
@@ -183,6 +184,7 @@ fi
 # one build (per commit) produces a documentation commit.
 #
 if [ "$(uname)" == "Darwin" -a "${TRAVIS_PULL_REQUEST}" != "false" -a -n "${JAZZY_ELIGIBLE}" ]; then
+  if [ "${TRAVIS_PULL_REQUEST_SLUG}" == "${TRAVIS_REPO_SLUG}" ]; then
```

Comment:
Is this changed related?

Reply:
Ah, thanks for spotting that.  It effectively compares the source repo to the destination repo, and only attempts to generate jazzy-doc if they are the same.

Because the Github credentials are defined on our repo, their values are missing when the Travis build runs against someone else's fork (and this also seems to apply to the PR build too).  Even if this were solved, we may not be permitted to push a docs commit to someone else's fork (depending on whether they selected the _Allow edits from maintainers_ option).

So, for now at least, we're restricted to running jazzy-doc updates from our own branches only.